packages/@lwc/integration-karma/test/component/lifecycle-callbacks/index.spec.js

```diff
@@ -269,6 +273,106 @@ describe('invocation order', () => {
     });
 });
 
+describe('connectedCallback/renderedCallback timing when reconnected', () => {
+    const scenarios = [
+        {
+            testName: 'shadow',
+            tagName: 'timing-parent',
+            Ctor: TimingParent,
+        },
+        {
+            testName: 'light',
+            tagName: 'timing-parent-light',
+            Ctor: TimingParentLight,
+        },
+    ];
+
+    scenarios.forEach(({ testName, Ctor, tagName }) => {
```

Comment:
minor nit: property order is different from the objects above in `scenarios`.

```suggestion
    scenarios.forEach(({ testName, tagName, Ctor }) => {
```